### PR TITLE
Port pieces plugin to Gtk3 and python3 and Deluge 2.x

### PR DESCRIPTION
**Pull request by Daniel-Marynicz**

### CODE REVIEW

**Comment by Daniel-Marynicz** [comment]
Current appearance on ubuntu 20.04 :)
Zaznaczenie_0139

**Comment by Daniel-Marynicz** [comment]
And i added Hover border and Selected border option.

Zaznaczenie_0140

**Comment by nicklan** [comment]
thanks for this! i'm trying to test it but getting deluge 2.0 on my machine is proving very difficult. 

it also doesn't seem to work now with deluge 1.x. i assume that's expected? how possible is it to get a plugin that works with 1.x and 2.x? ideally i'd like to maintain 1.x support until proper packaging is available for deluge 2.x (i have no idea why that is taking them so long, but until there is 1.x is still quite common)

**Comment by nicklan** [comment]
To clarify, I want to merge this for sure, but if we can't maintain 1.x compatibility we'll need some docs etc updates

**Comment by Daniel-Marynicz** [comment]
@nicklan I updated readme in last commit. And i propose release version 1.0 You should tag version 0.6 in github and release a .egg file for people who want to use this plugin on older linux systems.

File: pieces/multi_square.py [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+                color = self.colors.get_hover_border().get_color()
+                if self.is_selected(square):
+                    color = self.colors.get_selected_border().get_color()
+
+                cairoContext.fill_preserve()
+                cairoContext.set_line_width(4)
+                cairoContext.set_source_rgb(color.red_float, color.green_float, color.blue_float)
+                cairoContext.stroke()
+
+            x = x + 15
```
Comment (nicklan) [inline pieces/multi_square.py:227]: could we make the square size a variable (possibly even configurable) and get rid of all these hard-coded 15s? (i'd be fine with this in a follow-up change)

File: pieces/data/pieces_tab.glade [hunk 2]
```diff
@@ -1,120 +0,0 @@
-<?xml version="1.0"?>
```
[the remark this answers is not in the document]
Comment (Daniel-Marynicz) [reply pieces/data/pieces_tab.glade:1]: yes i will delete

File: pieces/multi_square.py (hunk 1 above)
Comment (Daniel-Marynicz) [reply pieces/multi_square.py:227]: ok i will  add in the config.ui option  and default value will be 15.

**Comment by Daniel-Marynicz** [comment]
@nicklan ok I added new options:
Zaznaczenie_0144

Example with very small setting:
Zaznaczenie_0141

default:
Zaznaczenie_0142

big:
Zaznaczenie_0143

**Comment by nicklan** [comment]
Great, this all seems to work, thanks much!

**Comment by bb010g** [comment]
Could a GitHub release with Python 3.6 egg artifacts be uploaded?

**Pull request merged**